helm/charts/nats/templates/configmap.yaml

```
@@ -254,6 +254,8 @@ data:
         {{- end }}
         }
       {{- end }}
+      {{- else }}
+      no_tls: {{ .Values.websocket.no_tls }}
```

Thanks a lot for the PR! Just fyi that I will change it to camel case to be consistent with the rest of the YAML fields:
```suggestion
      no_tls: {{ .Values.websocket.noTLS }}
```